CMakeLists.txt

```diff
@@ -299,16 +299,17 @@ add_custom_command(OUTPUT ${PROJECT_BINARY_DIR}/open62541.h
                    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/tools/amalgamate.py
                            ${GIT_COMMIT_ID} ${CMAKE_CURRENT_BINARY_DIR}/open62541.h ${exported_headers}
                    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/tools/amalgamate.py
-                           ${exported_headers}
-                           ${internal_headers})
```

Comment:
@jpfr I don't see the point why the generation of open62541.h is dependant on the internal_headers and lib_sources. If a c-file changes which has no impact on the external_headers, there is no need to regenerate the open62541.h. Or do I miss something?